### ISSUE TITLE
So what happened to this repository and project?

### ISSUE THREAD

**Issue by vongohren** [SmartIdentity/smartId-contracts#17]
It is alive, just found this http://www.deloitte.co.uk/smartid/ but according to this repo, and project owner, it is quite dead.
What is the status?

**Comment by supertyler** [comment]
@vongohren thanks for the note - this repo provides a reference implementation for a basic self sovereign id demo, but has not been updated in quite some time. Deloitte "Smart ID" work has extended to cover multiple ID protocol implementations (Sovrin, Uport, ERC 725/735 etc, which seem to be gaining momentum), and at this time it does not seem helpful to continue to push a parallel standard. This repo can be considered dormant. Happy to take any suggestions though.

**Comment by vongohren** [comment]
Yeah we are ourselves working with uPort, but always curious to what is going on.
But all these protocols still follow two working standards, that makes the possibility of interopability. So if you guys where to continue this work, I think you would like to adhear to https://w3c-ccg.github.io/did-spec/ & https://w3c.github.io/vc-data-model/
